pages/plans/[id].js

```diff
@@ -85,7 +85,7 @@ const PlanSummary = ({ planId, initialPlan, token }) => {
             await addAction(action);
             setShowAddAction(false);
           }}
-          officerName={goal.agreedWithName}
+          addedBy={getUsername(token)}
```

Comment:
Is renaming this to `addedBy` necessary do you think? Would be nice to keep the consistency with `residentName` and `officerName` if possible - plus the action hasn't been added at this point so the past-tense naming of `addedBy` is slightly confusing for me.

Reply:
(I agree with the naming for the API btw, just talking about this component)

Reply:
Makes sense!